### PR TITLE
A/B Test edit pass /1

### LINKED ISSUE #28536: Add A/B testing info for incremental migration
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6263e512-7395-7fd1-093c-6bf7dd719179
* Version Independent ID: 6263e512-7395-7fd1-093c-6bf7dd719179
* Content: [Incremental ASP.NET to ASP.NET Core Migration](https://learn.microsoft.com/en-us/aspnet/core/migration/inc/overview?view=aspnetcore-7.0)
* Content Source: [aspnetcore/migration/inc/overview.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/migration/inc/overview.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-migration**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

---
[Associated WorkItem - 67036](https://dev.azure.com/msft-skilling/Content/_workitems/edit/67036)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.learn.microsoft.com/en-us/aspnet/core/migration/inc/abtesting?view=aspnetcore-8.0&branch=pr-en-us-28551)

Fixes #28536

### CODE REVIEW

File: aspnetcore/migration/inc/abtesting.md [hunk 1]
```diff
@@ -10,17 +10,17 @@ ms.prod: aspnet-core
...
-This document will describe how to setup a conditional endpoint selection system to enable A/B testing during incremental migration. It assumes a setup as described [incremental migration overview](xref:migration/inc/overview) as a starting point.
+This document describes how to setup a conditional endpoint selection system to enable A/B testing during incremental migration. It assumes a setup as described in [incremental migration overview](xref:migration/inc/overview) as a starting point.
 
-## Conditional Endpoint Selection
+## Conditional endpoint selection
 
 To enable conditional endpoint selection, a few services need to be defined:
 
-1. Metadata that can be added to an endpoint (controllers, minimal APIs, etc) to turn on any conditional related logic. If this is not present on an endpoint, that endpoint will not incur any potential cost with checks that may be performed.
+1. Metadata that can be added to an endpoint to turn on any conditional related logic. Endpoints include controllers, minimal APIs, Razor Page, etc. If this metadata isn't present on an endpoint, that endpoint won't incur any potential cost with checks that may be performed.
```
Comment (Rick-Anderson) [inline aspnetcore/migration/inc/abtesting.md:23]: If this metadata isn't present on an endpoint, that endpoint won't incur any potential cost with checks that may be performed.
That's hard to parse. Can it be written simpler?

**Comment by Rick-Anderson** [comment]
@twsouthwick  hooked up to TOC and MS style guide fixes.

File: aspnetcore/migration/inc/abtesting.md (hunk 1 above)
Comment (twsouthwick) [reply aspnetcore/migration/inc/abtesting.md:23]: how about "If this metadata isn't added to an endpoint, no conditional checks will be performed for it."

**Review by twsouthwick — APPROVED** [review]

File: aspnetcore/migration/inc/abtesting.md (hunk 1 above)
Comment (Rick-Anderson) [inline aspnetcore/migration/inc/abtesting.md:23]:
```suggestion
1. Metadata that can be added to an endpoint to turn on any conditional related logic. Endpoints include controllers, minimal APIs, Razor Page, etc. If this metadata isn't added to an endpoint, no conditional checks are performed for it.
```
